app/pages/sections/routes.test.js

```diff
@@ -18,11 +18,13 @@ import * as commencementDateController from './commencement-date/controller';
 import * as serviceRecipientsController from './service-recipients/controller';
 import { getFundingSource } from '../../helpers/api/ordapi/getFundingSource';
 import { putFundingSource } from '../../helpers/api/ordapi/putFundingSource';
+import { getOrderDescription } from '../../helpers/api/ordapi/getOrderDescription';
```

Comment:
As above, I think we probably want to use the getOrderDescription that I've added as part of activity 8866 here, so this should be:

```javascript
import { getOrderDescription } from '../../helpers/routes/getOrderDescription';
```